### ISSUE TITLE
Release Ap-Nimbus v2.0.0

### ISSUE THREAD

**Issue by kwabenantim** [CardiacModelling/ap-nimbus#75]
### [appredict-docker](https://github.com/CardiacModelling/appredict-docker)

- [x] Update to [Chaste 2024.1](https://github.com/Chaste/Chaste/releases/tag/2024.1) and [ApPredict 2024.1](https://github.com/Chaste/ApPredict/releases/tag/v2024.1)
- [x] Upgrade base image to debian bullseye.
- [x] Build [appredict-chaste-libs](https://hub.docker.com/r/cardiacmodelling/appredict-chaste-libs):2.0.0 docker image.
- [x] Build [appredict-no-emulators](https://hub.docker.com/r/cardiacmodelling/appredict-no-emulators):2.0.0 docker image.
- [x] Build [appredict-with-emulators](https://hub.docker.com/r/cardiacmodelling/appredict-with-emulators):2.0.0 docker image.
- [x] Release repository v2.0.0

### [ap-nimbus-app-manager](https://github.com/CardiacModelling/ap-nimbus-app-manager)
- [x] Update to appredict-with-emulators:2.0.0 base image.
- [x] Upgrade to [Node 20](https://nodejs.org/en/blog/release/v20.15.0).
- [x] Build [ap-nimbus-app-manager](https://hub.docker.com/r/cardiacmodelling/ap-nimbus-app-manager):2.0.0 docker image.
- [x] Release repository v2.0.0

### [ap-nimbus-client](https://github.com/CardiacModelling/ap-nimbus-client)
- [x] Support Python 3.12
- [x] Build [ap-nimbus-client](https://hub.docker.com/r/cardiacmodelling/ap-nimbus-client/):2.0.0 docker image.
- [x] User Acceptance Testing
- [x] Release repository v2.0.0

### [ap-nimbus](https://github.com/CardiacModelling/ap-nimbus)
- [x] Update RtD documentation.
- [x] Update submodules for appredict-docker, app-manager, and ap-nimbus client to v2.0.0.
- [ ] Release repository v2.0.0

**Comment by kwabenantim** [comment]
> Does this relate to #76 @kwabenantim , can we get latest release to work with numpy 2 ?

This resolves #76 by installing `numpy>=1.24.4,<2.0`. I don't think [cellmlnamip](https://github.com/ModellingWebLab/cellmlmanip) currently supports numpy 2 thought - I'll add an issue for it.

**Comment by kwabenantim** [comment]
It looks like the numpy 2 errors [here](https://github.com/CardiacModelling/ap-nimbus-client/actions/runs/9681777837/job/26713158886) are pint-related (https://github.com/hgrecco/pint/issues/1974).

```
==================================== ERRORS ====================================
__________________ ERROR collecting files/tests/test_forms.py __________________
client/files/tests/test_forms.py:8: in <module>
    from files.forms import CellmlModelForm
client/files/forms.py:5: in <module>
    from cellmlmanip import load_model
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/cellmlmanip/__init__.py:3: in <module>
    from .main import load_model  # noqa
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/cellmlmanip/main.py:2: in <module>
    from cellmlmanip.parser import Parser
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/cellmlmanip/parser.py:15: in <module>
    from cellmlmanip.model import SYMPY_SYMBOL_DELIMITER, Model
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/cellmlmanip/model.py:15: in <module>
    from cellmlmanip.units import UnitStore
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/cellmlmanip/units.py:13: in <module>
    import pint
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pint/__init__.py:28: in <module>
    from .measurement import Measurement
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pint/measurement.py:12: in <module>
    from .quantity import Quantity
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pint/quantity.py:69: in <module>
    from .numpy_func import (
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pint/numpy_func.py:875: in <module>
    implement_single_dimensionless_argument_func(func_str)
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pint/numpy_func.py:864: in implement_single_dimensionless_argument_func
    func = getattr(np, func_str)
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/numpy/__init__.py:410: in __getattr__
    raise AttributeError("module {!r} has no attribute "
E   AttributeError: module 'numpy' has no attribute 'cumproduct'. Did you mean: 'cumprod'?
```
